src/js/components/TextInput/index.d.ts

```diff
@@ -14,13 +14,13 @@ export interface TextInputProps {
   onSelect?: ((x: { target: React.RefObject<HTMLElement>['current'], suggestion: any }) => void);
   onSuggestionsOpen?: ((...args: any[]) => any);
   onSuggestionsClose?: ((...args: any[]) => any);
-  placeholder?: string | React.ReactNode | JSX.Element;
+  placeholder?: string | React.ReactNode;
```

Comment:
Do we actually need the string type here?

Reply:
Yes now that I found about about the ReactNode we do not need string